pkg/cluster/init.go

```diff
@@ -10,18 +10,49 @@ import (
 	"encoding/base64"
 	"encoding/pem"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"math/big"
 	"net"
+	"net/http"
 	"os"
 	"path/filepath"
 	"time"
 
+	"github.com/mholt/archiver"
 	"github.com/pkg/errors"
 	"github.com/replicatedhq/kots/pkg/logger"
 	"go.uber.org/zap"
 )
 
+// Init will download all binaries for the cluster
+func Init(ctx context.Context, dataDir string) error {
```

Comment:
there's already a `clusterInit` function in the same package, this naming can be confusing, any reason not to have this be part of the other?

Reply:
maybe I'll rename `init` to `clientInit`?